packages/dynamics-lib/src/entities/__tests__/pocl-staging-exception.entity.spec.js

```diff
@@ -1,8 +1,8 @@
 import { PoclStagingException } from '../pocl-staging-exception.entity.js'
 import { Permission, retrieveGlobalOptionSets } from '../..'
 
-let optionSetData
 describe('pocl staging exception entity', () => {
+  let optionSetData
```

Comment:
After expanding this out again, looks like you've got multiple expect statements in this test file too that I'd missed in a previous review - please could you amend these to be separate tests; it's fine to have external functions that get you the test data (so a getSamplePOCLStagingException function, for instance).